Lib/fontTools/misc/plistlib.py

```diff
@@ -395,6 +395,19 @@ def totree(
     pretty_print=True,
     indent_level=1,
 ):
+    """Convert a value derived from a plist into an XML tree.
+
+    Args:
+        value: Any kind of value to be serialized to XML.
+        sort_keys: Whether keys of dictionaries should be sorted.
+        skipkeys: Keys which should be skipped when enumerating.
+        use_builtin_types: Whether to use standard plist types or
+            to wrap values in a Data block. Deprecated.
```

Comment:
`Data` should be formatted as a Python identifier.

This description is vague. It makes it sound like `use_builtin_types=False` will wrap _all_ values in `Data` blocks. The comment above `USE_BUILTIN_TYPES` gives more information. Also, this should explain what the default value of `None` means.

Reply:
yes, only binary data is wrapped